exercise/tests/test_models.py

```diff
@@ -93,6 +97,58 @@ def test_email_reply_scoring(self):
         self.assertTrue(score_one in email_reply.scores)
         self.assertFalse(score_two in email_reply.scores)
 
+    def test_exercise_specific_properties(self):
+        emails = EmailFactory.create_batch(5)
+        exercise = ExerciseFactory.create(emails=emails)
+        exercise_emails = ExerciseEmailProperties.objects.filter(exercise=exercise)
+        self.assertEqual(5, exercise_emails.count())
+
+        for cnt, email in enumerate(exercise_emails, 1):
+            email.web_page = ExerciseWebPage.objects.create(url="url_{}".format(cnt))
+            email.release_codes.add(
+                ExerciseWebPageReleaseCode.objects.create(
+                    release_code="Release Code {}".format(cnt)
+                )
+            )
+
+            if cnt == 4:
+                email.intercept_exercise = False
+            else:
+                email.intercept_exercise = True
+
+            email.save()
+
+        exercise_emails_updated = ExerciseEmailProperties.objects.filter(
```

Comment:
please use the method `exercise_specific_properties` instead to fetch the email properties